### PR TITLE
adding a template file to create an issue

### PR DESCRIPTION
**Pull request by Madhu-1**
the content of the template file is taken from heketi.
This template can be used to create an issue
or this can be used for any feature request.

Signed-off-by: Madhu Rajanna <mrajanna@redhat.com>

### CODE REVIEW

**Comment by Madhu-1** [comment]
@prashanthpai @aravindavk @kshlm please review

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+### Expected/desired behavior
+
+
+### Details on how to reproduce (minimal and precise)
+
+
+### Information about the environment:
+
+- Glusterd2 version used (e.g. v4.1.0 or master):
```
Comment (prashanthpai) [inline .github/ISSUE_TEMPLATE.md]: The users won't know how to get this information - they will have to go look elsewhere. It's more useful to ask users to run specific commands to get the information.

For example, ask users to provide: output of `glusterd2 --version` and `etcd --version`

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<!--
+This issue template is meant mainly for bug reports and feature requests.
+
+You can still use an issue to ask for help. In this case
+you should remove the template contents or use it as a guideline
+for providing information for debugging.
+-->
+
+### Kind of issue
```
Comment (prashanthpai) [inline .github/ISSUE_TEMPLATE.md]: This section isn't required at all. Feature requests do not follow this template. The contents of feature request can be very arbitrary in nature.

Further, we have labels - `bug` and `feature` that maintainers can set after triaging the issue

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+- Using External ETCD: (yes/no, if yes ETCD version):
+- If container, which container image: 
+- Using kubernetes, openshift, or direct install: 
+- If kubernetes/openshift, is gluster running inside kubernetes/openshift or outside: 
+
+
+### Other useful information
+
+* glusterd2 config file
+* glusterd2 log file
```
Comment (prashanthpai) [inline .github/ISSUE_TEMPLATE.md]: * glusterd2 log files from all nodes.
* Contents of `uuid.toml`

Also mention the default location of `uuid.toml`

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+* ETCD configuration
```
Comment (prashanthpai) [inline .github/ISSUE_TEMPLATE.md]: Another useful information is output of `/statedump` endpoint.

Comment (prashanthpai) [inline .github/ISSUE_TEMPLATE.md]: If it's an etcd issue, we'll likely need etcd logs from all nodes too

Comment (Madhu-1) [reply .github/ISSUE_TEMPLATE.md]: they might not be knowing the whether its ETCD issue or not, we can get it later if we think its an ETCD issue

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (Madhu-1) [reply .github/ISSUE_TEMPLATE.md]: done add one more section for it

File: .github/ISSUE_TEMPLATE.md (hunk 4 above)
Comment (Madhu-1) [reply .github/ISSUE_TEMPLATE.md]: done

**Pull request merged**